components/mpas-ocean/cime_config/buildnml

```diff
@@ -97,8 +97,8 @@ def buildnml(case, caseroot, compname):
             ic_prefix = 'oEC60to30v3wLI60lev.restart_theta_year26'
 
     elif ocn_grid == 'ECwISC30to60E1r2':
-        decomp_date = '200408'
-        decomp_prefix = 'mpas-o.graph.info.'
+        decomp_date = '230314'
+        decomp_prefix = 'partitions/mpas-o.graph.info.'
```

Comment:
@jonbob and @darincomeau, I'm opening this as a discussion PR so we can decide if having a subdirectory in `decom_prefix` will work before we move over to `E3SM`.